app/views/clients/_enrollment_table.haml

```diff
@@ -58,7 +58,8 @@
         %td
           %span{data: @client.program_tooltip_data_for_enrollment(e)}
             - if include_links
-              = link_to e[:project_name], project_path(id: e[:project_id])
+              - can_view_project_page = e[:project_name] != GrdaWarehouse::Hud::Project.confidential_project_name || current_user.can_edit_projects?
```

Comment:
Is include links something like `can_view_projects?`  I _think_ `can_view_project_page` should be something like:
```suggestion
              - can_view_project_page = current_user.can_edit_projects?
              - can_view_project_page ||=  e[:project_name] != GrdaWarehouse::Hud::Project.confidential_project_name && current_user.can_view_projects? 
```

Reply:
Your logic makes sense to me, and is easier to read, updated in https://github.com/greenriver/hmis-warehouse/pull/2267/commits/67df096dd4a687a20ffbb697e9e866140edab08e